README.md

```diff
@@ -57,6 +57,26 @@ Example
 Looking for example to use `react-flexbox-grid`? Head over to [react-flexbox-grid-example](https://github.com/roylee0704/react-flexbox-grid-example).
 
 
+Advanced composition
+-------
+Functions for generating Row and Column classNames are exported for use in other components.  For example, suppose you have a `MyFormInput` component that should also act as both a `Row` and a `Col`.
+
+```js
+import {getRowClassNames, getColClassNames} from 'react-flexbox-grid'
+
+export default function MyFormInput(props) {
+  return (
+    <form className={getRowclassNames(props)}>
+      <input className={getColClassNames(props)} />
```

Comment:
I don't think this example will work correctly, because `getRowClasssNames` and `getColClassNames` produce an **array**, but `className` requires a string. When I was playing with this, I got something like this:

```html
<form class=",row,around-xs">
```

I think it relies on `createProps` to concatenate it to a valid class name.